src/classes/HH_Container_LCTRL.cls

```diff
@@ -37,6 +37,28 @@
 public class HH_Container_LCTRL {
     
     /*******************************************************************************************************
+    * @description trick the packaging spider to pickup references to custom labels only used by our
+    * lightning components.
+    */
+    public static void packageLabels() {
```

Comment:
Yes, but this list does not contain even every NPSP label you reference.  For example, `stgBtnCancel` or `bdiBtnClose`.  It also contains references to labels you don't use, like `lblStreet` (you use a label just called `Street`).